### PR TITLE
Remove python 2 and update numpy/pandas version

### PR DESCRIPTION
**Pull request by kyogesh**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -17,9 +17,8 @@
 
 
 # Check this Python version is supported
-if any([v < (2, 6), (3,) < v < (3, 7)]):
-    raise Exception("Unsupported Python version %d.%d. Requires Python >= 2.7 "
-                    "or >= 3.7." % v[:2])
+if v > (3, 7):
```
Comment (franciscofabian) [inline setup.py:20]:
```suggestion
if v < (3, 7):
```

File: setup.py [hunk 2]
```diff
@@ -59,40 +58,29 @@ def read(*parts):
 optional_libs = ['numexpr>=2.6.9']
 
 install_requires = [
-    'azure-storage-blob==12.3.2;python_version=="2.7"',
     'azure-storage-blob>=12.4.0;python_version>="3.7"',
```
Comment (franciscofabian) [inline setup.py:61]: We no longer need all this `;python_version>="3.7"` everywhere, we can remove them.

File: setup.py [hunk 3]
```diff
@@ -112,8 +100,6 @@ def read(*parts):
     'Topic :: Software Development :: Libraries :: Python Modules',
     'Operating System :: Microsoft :: Windows',
     'Operating System :: Unix',
-    'Programming Language :: Python :: 2',
-    'Programming Language :: Python :: 2.7',
     'Programming Language :: Python :: 3',
     'Programming Language :: Python :: 3.7',
     'Programming Language :: Python :: 3.8',
```
Comment (franciscofabian) [inline setup.py:105]: Can you remove 3.8 and 3.9, and add 3.11?

File: setup.py (hunk 1 above)
Comment (ffinkbe) [reply setup.py:20]: Actually we need 3.11 compatibility `if v < (3, 11):`

**Comment by franciscofabian** [comment]
We also need to bump the version number to create a new version, like here https://github.com/visualfabriq/parqueryd/commit/b4552078019bb94b852d6cc55a212bb38eca2a68, probably 1.1.0 would be good

**Comment by franciscofabian** [comment]
I have just realized that this branch is based from `uat` instead of `master`, we need it to be based on master

**Pull request closed without merging**